main.py

```diff
@@ -1,10 +1,48 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        data = json.load(file)
+
+    for player_nickname, info in data.items():
+        if not Race.objects.filter(name=info["race"]["name"]).exists():
+            Race.objects.create(
+                name=info["race"]["name"],
+                description=info["race"]["description"]
+            )
+        race = Race.objects.get(name=info["race"]["name"])
```

Comment:
if you create an object you do 3 query
I recommend you use Model.objects.get_or_create method

Reply:
Ok, good advice.